lib/flex_commerce_api/json_api_client_extension/requestor.rb

```diff
@@ -35,10 +35,9 @@ def resource_path(parameters, record = nil)
       end
 
       def encoded(part)
+        # Do not encode the slug
```

Comment:
Should this comment still be here ?  There doesnt appear to be any code backing it up ?